dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobConfiguration.java

```diff
@@ -293,6 +294,9 @@ public boolean hasCronExpression() {
         @JsonSubTypes.Type(
             value = SingleEventDataSynchronizationJobParameters.class,
             name = "SINGLE_EVENT_DATA_SYNC"),
+        @JsonSubTypes.Type(
+            value = TrackerDataSynchronizationJobParameters.class,
+            name = "TRACKER_DATA_SYNC"),
```

Comment:
I think we can rename this to TRACKED_ENTITY_DATA_SYNC

Which means all data associated with the tracked entity (including tracker programs).